styles.css

```diff
@@ -0,0 +1,219 @@
+/* ==============
+   UTILITY STYLES
+   ============== */
+
+/* Corrects the layout around floated elements */
+.u-clearfix::before,
+.u-clearfix::after {
+  content: " ";
+  display: table;
+}
+
+.u-clearfix::after {
+  clear: both;
+}
+
+/* Displays list items as inline block  */
+.u-list-inline li {
+  display: inline-block;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+
+body {
+  background-color: #FCFCFC;
+  color: var(--color-slate);
+  font-family: 'Crimson Text', serif;
+  font-size: 9px;
+  line-height: 1.3;
+  margin: 0;
+}
+
+.donut-img {
+  position: relative;
+  top: -20px;
+  left: -75px;
+}
+
+a {
+  color: rgb(29, 38, 98);
+}
+
+h2 {
+  color: rgb(29, 38, 98);
+  position: absolute;
+  top: 25px;
+  right: 15px;
+  font-size: 20px;
+  font-family: 'Arvo', serif;
+}
+
+span {
+  color: rgb(255, 50, 0);
+  font-family: 'Dancing Script', cursive;
+  font-size: 25px;
+  position: absolute;
+  top: -22px;
+}
+
+
+.spicerack {
+  width: 60px;
+  position: absolute;
```

Comment:
In general, throughout this layout, I'm seeing too many absolutely positioned elements. Absolute positioning should be used as a last resort and only for a few things in a layout, ideally. If it would be helpful to see an example of other ways to achieve this same layout without absolute positioning, let me know?